lib/nexmo_developer/app/models/blog/blogpost.rb

```diff
@@ -36,10 +36,17 @@ def self.build_blogpost_from_path(path, locale)
 
     default_not_found_page(path) unless File.exist?(path)
 
-    document = File.read(path).gsub('/content/blog/') do |match| # gsub Netlify img urls
+    # gsub Netlify - img urls to S3 Bucket
+    document = File.read(path).gsub('/content/blog/') do |match|
       "#{Blog::Blogpost::CLOUDFRONT_BLOG_URL}blogposts/#{match.gsub('/content/blog/', '')}"
     end
 
+    # gsub Netlify - embedded YOUTUBE Video
+    document = document.gsub(/<youtube id=\"(\w+)\"><\/youtube>/) do |match|
+      youtube_id = match[/(?<=\").*(?=\")/]
+      "<center class='video'><br><iframe width='448' height='252' src='https://www.youtube-nocookie.com/embed/#{youtube_id}' frameborder='0' allow='accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture' allowfullscreen></iframe><br><br></center>"
```

Comment:
I am just not sure if this width and height would work as the previous blog post because they used to take the entire width of the page if I am not mistaken.

Reply:
Everything's already responsive - those value are there just to make the element a `block` element